### DISCUSSION TITLE
GitHub trademark - use as button icon

### DISCUSSION THREAD

**Discussion by eTee15** [community/community#21827] [New to GitHub]
<p>What are the jurisdictions and policies regarding usage around GitHub’s icons when using them in a web app? I know other icon platforms like Flaticon require you to credit the author in your code but Font Awesome 5 doesn’t say (at least that I know of). For my app, I’m using the icon as a button with a link that opens up to my GitHub profile. I have contacted GitHub just now but I would like to know here in case GitHub doesn’t respond in time (long story short: this web app is for a university project due in couple months).</p>
<p>Thanks!</p>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/maureento8888">@maureento8888</a>,</p>
<p>According to <a href="https://github.com/logos" rel="nofollow noopener">GitHub’s logo policy</a>, this is absolutely fine.</p>

**Comment by eTee15** [comment]
<p>Wow, I didn’t find that page LOL. Thanks a ton!</p>
